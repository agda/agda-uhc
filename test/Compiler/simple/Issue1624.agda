module Issue1624 where

open import Common.Char
open import Common.String
open import Common.List

test : List Char → String
test ('0' ∷ ('x' ∷ xs)) = "x"
test ('0' ∷ ('b' ∷ xs)) = "b"
test xs               = "f"


open import Common.IO
open import Common.Unit

main : IO Unit
main = putStr (test x)
  where x = '0' ∷ ('x' ∷ ('1' ∷ []))
