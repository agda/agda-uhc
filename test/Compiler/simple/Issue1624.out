EXECUTED_PROGRAM

ret > ExitSuccess
out > x
